wallet/macros/src/wallet/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod client;
+pub mod server;
+// pub mod wasm;
```

Comment:
Is this intentional that wasm is commented out?

Reply:
Adding a `//TODO` note.  This is a placeholder with some code copied over from `RPC` meant to automate bindings for `WalletApi`.  I started working on bindings but quickly concluded that the `WalletApi` needed to be stabilized before the bindings were made.  

Also, there have been efforts made by @starkbamse and other contributors with regard to improving types in RPC bindings (which also apply here) and I had to freeze the effort till January in order to get a concise view of all issues in front of us so that all of this can be addressed and implemented as a single dedicated effort/pass.